CONTRIBUTING.md

```diff
@@ -187,16 +188,18 @@ This is a short way for you to say that you are entitled to contribute the patch
 It is a legal statement that asserts your agreement with the [DCO](#developers-certificate-of-origin).
 Therefore, it must have your *real name* (First Last) and a valid e-mail address.
 Adding this tag can be done automatically by using `git commit -s`.
-If you are editing files and committing through GitHub, you must write your real name in the “Name” field in GitHub settings and the email used in the "Signed-off-by:" must be your primary github email.
-You must manually add the "Signed-off-by:" to the commit message that github requests.
+If you are editing files and committing through GitLab, you must write your real name in the “Full Name” field in your GitLab profile and the email used in the "Signed-off-by:" must be your "Commit email" address.
```

Comment:
I know this originally was Github and thus you changed to Gitlab, but all other places refer to it as lower-case gitlab. better stick to one (lower/upper case the G for all occurrences) 
```suggestion
If you are editing files and committing through GitLab, you must write your real name in the “Full Name” field in your GitLab profile, and the email used in the "Signed-off-by:" must be your "Commit email" address.
```